src/Catalyst.Core.Modules.Web3/Controllers/EthController.cs

```diff
@@ -36,35 +35,37 @@ namespace Catalyst.Core.Modules.Web3.Controllers
     [Route("api/[controller]/[action]")]
     public class EthController : Controller
     {
+        private static readonly object Lock = new object();
         private readonly IWeb3EthApi _web3EthApi;
-        private readonly IEthRpcService _ethRpcService;
         private readonly IJsonSerializer _jsonSerializer;
         private readonly IWeb3HandlerResolver _handlerResolver;
         private readonly ILogger _logger = Log.Logger.ForContext(typeof(EthController));
 
-        public EthController(IWeb3EthApi web3EthApi, IWeb3HandlerResolver handlerResolver, IJsonSerializer jsonSerializer, IEthRpcService ethRpcService)
+        public EthController(IWeb3EthApi web3EthApi, IWeb3HandlerResolver handlerResolver, IJsonSerializer jsonSerializer)
         {
-            _ethRpcService = ethRpcService;
             _web3EthApi = web3EthApi ?? throw new ArgumentNullException(nameof(web3EthApi));
             _handlerResolver = handlerResolver ?? throw new ArgumentNullException(nameof(handlerResolver));
             _jsonSerializer = jsonSerializer ?? throw new ArgumentNullException(nameof(jsonSerializer));
         }
 
-        [HttpGet]
-        public long? eth_blockNumber() { return _ethRpcService.eth_blockNumber(); }
-
         [HttpPost]
         public JsonRpcResponse Request([FromBody] JsonRpcRequest request)
         {
-            _logger.Information("ETH JSON RPC request {id} {method} {params}", request.Id, request.Method, request.Params);
             EthWeb3RequestHandlerBase handler = _handlerResolver.Resolve(request.Method, request.Params.Length);
             if (handler == null)
             {
-                return new JsonRpcErrorResponse {Result = null, Error = new Error {Code = (int) ErrorType.MethodNotFound, Data = $"{request.Method}", Message = "Method not found"}};
+                JsonRpcErrorResponse errorResponse = new JsonRpcErrorResponse {Result = null, Error = new Error {Code = (int) ErrorType.MethodNotFound, Data = $"{request.Method}", Message = "Method not found"}};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 211 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)